### ISSUE TITLE
Не отражаются формы объектов конфигурации под linux.

### ISSUE THREAD

**Issue by kamaxmax** [1C-Company/1c-edt-issues#343]
### Описание ошибки

При попытке просмотра любой формы 
![2022-02-02_11h47_42](https://user-images.githubusercontent.com/13096281/152123203-26730484-9451-431d-812c-e6fbf9d0abcc.png)
объекта конфигурации не отображаются элементы формы. См. скриншот.

### Как воспроизвести

Просто открыть любую форму объекта конфигурации.

### Ожидаемое поведение

Форма должна отражать все визуальные элементы.

### Лог рабочей области

[Untitled.log](https://github.com/1C-Company/1c-edt-issues/files/7984936/Untitled.log)

### Версия 1С:EDT

Ruby 2021.2.12; Ruby 2021.3.1

### Операционная система

Linux

**Comment by vadimeg** [comment]
Сообщите версию платформы, пожалуйста

**Comment by vadimeg** [comment]
Можете проверить на новой конфигурации? Создать новую конфигурацию, справочник и форму справочника. Тоже не открывается?

**Comment by kamaxmax** [comment]
От платформы не зависит, пробовал на разных, например: 8.3.19.1229 и 8.3.20.1623. Да, проверял на пустой конфигурации, с добавлением одного объекта - та же картина.

**Comment by vadimeg** [comment]
А какой именно линукс? И версия

**Comment by kamaxmax** [comment]
Ubuntu 20.04

**Comment by EChudnovsky** [comment]
Дополню описание проблемы.
Подобное поведение наблюдается только под этой учёткой.

Рабочая станция- это:

1. Linux Ubuntu 20.04
2. x2go качестве клиента удаленных приложений
3. EDT (оффлайн) как общая установка для всех разработчиков

**Comment by MaksimDzyuba** [comment]
Здравствуйте, а не могли бы на машине своей поискать файлы вот по такой маске имени "hs_err_pid*" и выслать один из них нам, пожалуйста, это очень поможет понять, что не так в текущей операционной системе.

**Comment by kamaxmax** [comment]
[hs_err_pid1645370.log](https://github.com/1C-Company/1c-edt-issues/files/8014668/hs_err_pid1645370.log)

**Comment by EChudnovsky** [comment]
Вот данные еще одного профиля в котором начались проблемы с отражением форм
[hs_err_pid120722.log](https://github.com/1C-Company/1c-edt-issues/files/8015211/hs_err_pid120722.log)

**Comment by MaksimDzyuba** [comment]
нет, к сожалению не те файле, в них нет следа проблемы, а их вообще много, если нет, то можно их все в архив и прислать, пожалуйста, чтобы я нашел нужный с проблемой

**Comment by EChudnovsky** [comment]
> нет, к сожалению не те файле, в них нет следа проблемы, а их вообще много, если нет, то можно их все в архив и прислать, пожалуйста, чтобы я нашел нужный с проблемой

Вот еще данные (свежие :) ) из другого профиля в котором начались проблемы с отображением форм
[hs_err_pid120722.log](https://github.com/1C-Company/1c-edt-issues/files/8015277/hs_err_pid120722.log)

**Comment by vadimeg** [comment]
Ошибка происходит внутри JRE при работе кода из JavaFX graphics. JavaFX используется в формах для отображения форматированного текста в палитре свойств. Ошибка довольно распространенная на linux при работе с GTK. Как воспроизвести не понятно, как исправить пока тоже. Для начала можно попробовать проверить версию GLIBC (ldd --version) и, если меньше 2.27, переустановить. Также можно попробовать обновить GTK3, JDK.

**Comment by MaksimDzyuba** [comment]
Здравствуйте, а скажите, пожалуйста, мы перешли на новый eclipse, обновляли компоненты по генерации лайоута для форм, может это косвено решило проблему, в 2022.1?

**Comment by Kellendros007** [comment]
Хочу внести свои 5 копеек по проблеме отрисовки форм, у меня наблюдается такая проблема
![image](https://user-images.githubusercontent.com/32325196/192165942-1c694f98-7501-479d-b9f6-9a65bada997f.png)
![image](https://user-images.githubusercontent.com/32325196/192166303-ff97e8b1-eeeb-423f-8afc-2e5b5eae71b6.png)
а так же
![image](https://user-images.githubusercontent.com/32325196/192166438-c2a3a773-c624-4a09-998d-928789f17259.png)

при этом я пробовал 
EDT 2021.3.4/2022.1.2
Ubuntu 20.04.5/22.04.1 (со всеми обновлениями)
Liberica jdk/jre 11.0.16.1+1/11.0.12+7
Конфигурация УТ 11.4.13.275/пустая конфигурация

Платформа 8.3.20.1674

**Comment by vadimeg** [comment]
По изначальной проблеме в линуксе с JavaFX. Удалось попробовать на новой версии EDT? Если да, то есть ли изменения?
Предлагается также убрать тэг критичности, т.к. формы открываются с ключем -DnativeFormLayoutCalculation=false

**Comment by vadimeg** [comment]
Закрываю баг, т.к. воспроизвести не удалось. Доп. информации тоже нет. Если будет воспроизводиться, то просьба переоткрыть

**Comment by JohnLarin** [comment]
2024.2.2
Проблема сохраняется до сих пор. Помогает только ключ -DnativeFormLayoutCalculation=false
